eta/core/storage.py

```diff
@@ -613,6 +614,47 @@ def delete(self, file_or_folder_id):
         self._service.files().delete(
             fileId=file_or_folder_id, supportsTeamDrives=True).execute()
 
+    def delete_all_files_in_folder(self, folder_id):
+        '''Deletes all the files in the Google Drive folder and retains the
+        original folder.
+
+        Args:
+            folder_id: the ID of the Google Drive folder from which the files
+                are to be deleted
+
+        Returns:
```

Comment:
The function doesn't actually return anything.  In such as case, you can omit the "Returns" section of the doc string.

Reply:
removed it, thanks.